addons/hw_drivers/main.py

```diff
@@ -31,6 +31,7 @@
 drivers = []
 interfaces = {}
 iot_devices = {}
+ssh_service = True
```

Comment:
I am a bit lost as of why it is initiated to `True` we assume it always run by default? So if we deactivate it but restart the IoT box it will be activated ?

Reply:
No, the re-activation only append if the send_all devices receive the ssh_service as True